decidim-accountability/lib/decidim/accountability/result_serializer.rb

```diff
@@ -18,11 +19,11 @@ def serialize
           id: result.id,
           category: {
             id: result.category.try(:id),
-            name: result.category.try(:name)
+            name: result.category.try(:name) || empty_translatable
```

Comment:
why use Rails' `try` when current Ruby version already has `&.`?

Reply:
`try` allows calling methods even if the object doesn't respond to them (which could happen if the category is nil):

<img width="414" alt="image" src="https://user-images.githubusercontent.com/5254/52946921-88c03980-3375-11e9-90ba-d1bb3da260a7.png">

Reply:
Thanks for your answer :D Yes (I knew) but this is the same as accepting that we don't know with which kind of object we are dealing. I'm more for "fail fast" than "I don't care".

Reply:
Well, it's mainly used for nil values (categories can be optional).